app/helpers/dwc_occurrences_helper.rb

```diff
@@ -0,0 +1,41 @@
+module DwcOccurrencesHelper 
+
+  def dwc_occurrences_metadata(dwc_occurrences = nil, project_id: nil)
+    records = dwc_occurrences
+    project_id ||= sessions_current_project_id
+
+    # Anticipate multiple sources of records in future (e.g. AssertedDistribution)
+    records ||= DwcOccurrence.where(project_id: project_id).where(dwc_occurrence_object_type: 'CollectionObject').all
+
+    data = CollectionObject.where(project_id: project_id)
+
+    # without occurrences!
+    #
+
+    a = {
+      data: {
+        total_collection_objects: data.count,
+        enumerated_objects: data.sum(:total),
+        freshness: {
+          never: data.left_outer_joins(:dwc_occurrence).where(dwc_occurrences: {id: nil}).count,
+          one_day: data.where("updated_at > ?", 1.day.ago).count,
```

Comment:
On this one but also those having similar pattern: `.where(updated_at: 1.day.ago..)`

```ruby
> puts CollectionObject.where(project_id: 42).where(updated_at: 1.day.ago..).to_sql
SELECT "collection_objects".* FROM "collection_objects" WHERE "collection_objects"."project_id" = 42 AND "collection_objects"."updated_at" >= '2021-06-06 15:49:41.657006'
```

*(`>=` instead of `>` but given the time resolution this hardly makes a difference)*